Jenkinsfile

```diff
@@ -1,5 +1,13 @@
-buildPlugin(useContainerAgent: true,
-            configurations: [
-              [platform: 'linux', jdk: '17'],
-              [platform: 'windows', jdk: '11'],
+/*
+ See the documentation for more options:
+
+https://github.com/jenkins-infra/pipeline-library/
+
+*/
+buildPlugin(
+  useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
+  configurations: [
+    [platform: 'linux', jdk: 21],
+    [platform: 'linux', jdk: 11],
```

Comment:
Why the discrepancy from archetype?

Reply:
It's because of me. I didn't see that the archetype didn't require Java 11 testing anymore. I'll change that.

Reply:
it does not harm - some would say this is a good thing™️.

Reply:
I tend to agree. If we say plugin xxxx still supports java 11, it's probably better to still have a build to test with it...